draft-dekater-scion-controlplane.md

```diff
@@ -1461,9 +1472,77 @@ When the segment-request handler of a *core AS* control service receives a path
 
 # Security Considerations
 
-One of the fundamental objectives that guided the design of SCION is security, in particular network security. See chapter 7 of the SCION book (Security Analysis), which states the precise security goals of various network participants and how SCION achieves these goals in the presence of different types of adversaries {{CHUAT22}}.
+This section describes the possible security risks and attacks that SCION's control plane may be prone to, and how these may be mitigated. The focus lies on *inter*-AS routing: SCION does not solve intra-AS routing issues, nor does it provide end-to-end payload encryption, and identity authentication. These topics lie therefore outside the scope of this section.
+
+As described previously, the goal of SCION’s beaconing process is to discover paths between any two ASes and make them available to other ASes. This section examines several possible approaches to manipulating the beaconing process in the SCION control plane, and shows for each case to what extent SCION's design can prevent the corresponding attack or help to mitigate it.
+
+- Path hijacking through interposition (see [](#path-hijack))
+- Creation of spurious ASes (see [](#fake-ases))
+- Peering link misuse (see [](#peer-link-misuse))
+- Manipulation of the path selection process (see [](#manipulate-selection))
+
+**Note:** This section only discusses SCION control plane- and routing-specific security considerations. For security considerations related to the SCION control-plane PKI, see {{I-D.scion-cppki}}. {{I-D.scion-dp}} includes security considerations that concern the SCION data plane and data forwarding.
+
+
+## Path Hijacking through Interposition {#path-hijack}
+
+To attract traffic and to include its own AS in segments and paths, an adversary might try to manipulate the beaconing process. There are several imaginable options for a malicious AS M to do this:
+
+- By blocking the traffic between A and B, and thus force traffic redirection through M.
+- By intercepting and disseminating a PCB on its way from AS A to neighbor AS B, and injecting its (AS M's) own hop fields into the PCB toward downstream ASes. The goal is to offer AS B an attractive up-segment that traverses AS M to the core.
+- By modifying the hop fields of an already existing path, in order to interpose the own AS in the path.
+
+The first attack is fundamental and generally cannot be prevented. But SCION is able to mitigate the other two attacks: The second attack is detectable by downstream ASes, because a PCB disseminated by AS A towards AS B contains an identifier for the egress interface from AS A to AS B, protected by A's signature. This will cause verification of the manipulated inbound PCBs to fail, as the adversary's PCBs cannot contain A's correct signature. The third attack is made impossible by the hop field's MAC, which protects the hop field's integrity and chains it with the previous hop fields on the path.
```

Comment:
I think the first attack might deserve a little more explanation:
* How could charlie block the Alice-Bob connection?
* Why would the traffic be re-directed to Charlie, without Charlie being already in a privileged position?
* Why is it that we can't prevent it?
I personally do not have the answers.

Reply:
@jiceatscion, I think you have a fair point. But unfortunately, I also have no answers. I took it from the SCION book, and given the fact that there was no explanation there, I thought it was obvious for all people with an IT background...

Reply:
This is coming from section 7.4.1 Path Hijacking through Interposition. These attacks requires the adversary to be on-path (which is a very strong assumption).  I clarified this in the text and reworded. Also the last attach could not be prevented, but the different path would be immediately visible to endpoints.